### PR TITLE
add[resource card]:master dataviz file direct download

### PR DESCRIPTION
**Pull request by jeanservaas**
Closes https://github.com/carbon-design-system/carbon-website/issues/1563

- [x] Updated Chart anatomy images (desktop and mobile) with new zoom bar and tooltip
- [x] Added master data vis design file direct download resource card to data viz > getting started page
- [x] Added master data vis design file direct download resource card to resources page
- [x] Fixed incorrect donut chart guidance on legend page
- [x] Updated some palettes on legend page that didn't correspond with our docs

### CODE REVIEW

**Comment by theiliad** [comment]
🎉  🎉  You did it! this is great

File: src/pages/data-visualization/chart-anatomy/index.mdx [hunk 1]
```diff
@@ -86,17 +86,17 @@ Circular charts are primarily pie and donut charts. Other circular layout charts
...
+    5. Graph frame<br/>
+    6. Big number<br/>
   </Aside>
 </Column>
 </Row>
 
 #### Labels
 
-In a circular chart, labels offer the percentage value of the whole for an individual category.
+In a circular chart, labels offer the percentage value of the whole for an individual category. When the graphic translation of the data is less than 3 degrees, a callout is used to clearly associate the label with the slice. If the data translates as less than 1 degree, a slice will not be rendered on the chart—the data would not be keyboard accessible or available in a tooltip either. The only way to see the data in this scenario would be via a data table, a feature that we'd like to implement for all charts to enhance accessibility.
```
Comment (theiliad) [inline src/pages/data-visualization/chart-anatomy/index.mdx:99]:
```suggestion
In a circular chart, labels offer the percentage value of the whole for an individual category. When the graphic translation of the data is less than 3 degrees, a callout is used to clearly associate the label with the slice. If the data translates as less than 1 degree, a slice will not be rendered on the chart, the data would not be keyboard accessible or available in a tooltip either. The only way to see the data in this scenario would be via a data table, a feature that we'd like to implement for all charts to enhance accessibility.
```

Comment (theiliad) [reply src/pages/data-visualization/chart-anatomy/index.mdx:99]: is that `—` correctly in there?

**Comment by jeanservaas** [comment]
@aagonzales what do you think. If we leave it in box I should add (IBM Internal link) like the other one.

**Comment by aagonzales** [comment]
Yeah I think adding that label would suffice for now.

**Comment by theiliad** [comment]
> @aagonzales what do you think. If we leave it in box I should add (IBM Internal link) like the other one.

does Box do version control? git might be better

**Comment by theiliad** [comment]
What’s stopping us from going live on github right now? Is there a process?
Let’s put this in the hands of everyone rather than just an internal link

**Pull request merged**